ddtrace/llmobs/_integrations/bedrock.py

```diff
@@ -36,9 +34,7 @@ def _llmobs_set_tags(
         operation: str = "",
     ) -> None:
         """Extract prompt/response tags from a completion and set them as temporary "_ml_obs.*" tags."""
-        if span.get_tag(PROPAGATED_PARENT_ID_KEY) is None:
-            parent_id = _get_llmobs_parent_id(span) or "undefined"
-            span._set_ctx_item(PARENT_ID_KEY, parent_id)
+        LLMObs._instance._activate_llmobs_span(span)
```

Comment:
Note: this is an existing issue and we'll need to fix it after this PR. But because of the way the bedrock integration uses shared tracing handlers, we don't actually get to reuse the `BaseLLMIntegration.trace()` helper to activate the span immediately when it's started. We'll need to fix it to get everything context activation-wise in one place.